### PR TITLE
[SPARK-37308][SQL][TESTS] Partially revert SPARK-37214's test due to flakiness

### PR DESCRIPTION
**Pull request by HyukjinKwon**
### What changes were proposed in this pull request?

This PR reverts one of the tests added at https://github.com/apache/spark/pull/34490 which is (pretty seriously) flaky.

### Why are the changes needed?

Other tests seem passing fine so it's likely test specific issue. so only test is proposed to be reverted for now.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

CI in this PR should test this out.

### CODE REVIEW

File: sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala [hunk 1]
```diff
@@ -359,9 +359,6 @@ class DDLParserSuite extends AnalysisTest with SharedSparkSession {
       false,
       LocalTempView)
     comparePlans(parsed2, expected2)
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala:361]:
```suggestion
    comparePlans(parsed2, expected2)

    // TODO(SPARK-37367): Reeable exception test in DDLParserSuite.create view -- basic
    // val v3 = "CREATE TEMPORARY VIEW a.b AS SELECT 1"
    // intercept(v3, "It is not allowed to add database prefix")
```

File: sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala [hunk 2]
```diff
@@ -360,8 +360,9 @@ class DDLParserSuite extends AnalysisTest with SharedSparkSession {
...
 
-    val v3 = "CREATE TEMPORARY VIEW a.b AS SELECT 1"
-    intercept(v3, "It is not allowed to add database prefix")
+    // TODO(SPARK-37367): Reeable exception test in DDLParserSuite.create view -- basic
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala:363]:
```suggestion
    // TODO(SPARK-37367): Reenable exception test in DDLParserSuite.create view -- basic
```

**Review by cloud-fan — APPROVED** [review]

**Comment by cloud-fan** [comment]
This seems like a github action issue, that the code being tested did not include the parser change that checks temp view and throws exception.

Let's comment it for now and investigate it later.

**Comment by HyukjinKwon** [comment]
Yeah .. I also hardly believe that the test has an actual issue.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM for now. I agree with the decision. Thanks!

**Comment by HyukjinKwon** [comment]
Let me merge this. The only relavent testing here should be just linters that already has passed.

**Comment by HyukjinKwon** [comment]
Merged to master and branch-3.2.

**Pull request closed without merging**